File: 2017/iOS/aulas/aula02/objc/MeusDados OBJC/MeusDados OBJC/ViewController.m
//
//  ViewController.m
//  MeusDados OBJC
//
//  Created by Lucas Rocha on 01/03/17.
//  Copyright © 2017 Lucas Tavares. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    // Insere os textos iniciais nas labels
    meuLabel1.text = @"Meu nome é ...";
    meuLabel2.text = @"Minha idade é ...";
    meuLabel3.text = @"Minha cidade é ...";
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)exibir:(id)sender {
    
    // Substitui o valor das Labels quando o botão "Exibir" é pressionado.
    meuLabel1.text = @"Lucas Tavares";
    meuLabel2.text = @"19 anos";
    meuLabel3.text = @"São Paulo";
}

- (IBAction)limpar:(id)sender {
    
    // Limpa o valor das Labels quando o botão "Limpar" é pressionado.
    meuLabel1.text = @"";
    meuLabel2.text = @"";
    meuLabel3.text = @"";
}
@end

File: iOS/aulas/aula02/swift/MeusDados Swift/MeusDados Swift/ViewController.swift
//
//  ViewController.swift
//  MeusDados Swift
//
//  Created by Lucas Rocha on 01/03/17.
//  Copyright © 2017 Lucas Tavares. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var myNameLabel: UILabel!
    @IBOutlet weak var myAgeLabel: UILabel!
    @IBOutlet weak var myCityLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        // Atualiza o texto das Labels
        myNameLabel.text = "Meu nome é:"
        myAgeLabel.text  = "Minha idade é:"
        myCityLabel.text = "Minha cidade é:"
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


    @IBAction func show(_ sender: Any) {
        // Exibe as informações nas Labels
        myNameLabel.text = "Meu nome é: Lucas"
        myAgeLabel.text  = "Minha idade é: 19"
        myCityLabel.text = "Minha cidade é: São Paulo"
    }
    
    @IBAction func hide(_ sender: Any) {
        // Esconde as informações nas Labels
        myNameLabel.text = "Meu nome é:"
        myAgeLabel.text  = "Minha idade é:"
        myCityLabel.text = "Minha cidade é:"
    }
    
}

